app/Services/GetPreliminaryInvoice.php

```diff
@@ -212,7 +212,7 @@ public function getBill(string $start_date, string $end_date, $smartMe, $price_a
 
         $countOfAllDaysInMonhtsInvolved = 0;
         foreach($months as $month) {
-            $countOfAllDaysInMonhtsInvolved = $countOfAllDaysInMonhtsInvolved + Carbon::createFromDate('2022',$month, 1)->daysInMonth;
+            $countOfAllDaysInMonhtsInvolved = $countOfAllDaysInMonhtsInvolved + Carbon::createFromDate(now()->year, $month, 1)->daysInMonth;
```

Comment:
@negoziator 
Ha! Ja, den var ikke gået ret længe endnu - glimrende eksempel på, når noget går for stærkt. Jeg burde nok give kvajebajer på den der :laughing:

Reply:
haha .. Det er hvad der sker 💯